### ISSUE TITLE
PHP Warning:  DirectorUtils::decode(): Cannot assign to an array of nodes (duplicate subnodes or attr detected) in web/classes/Utils.php on line 48

### ISSUE THREAD

**Issue by davidbarratt** [kyxucf/kyxucf#1]
I'm getting the following PHP Error on every page request:

```
PHP Warning:  DirectorUtils::decode(): Cannot assign to an array of nodes (duplicate subnodes or attr detected) in web/classes/Utils.php on line 48
```

Thanks!

**Comment by john-henry** [comment]
Do you have a ref for what was fixed between 1.2.1 and 1.2.2? Running into same error on old site.

**Comment by davidbarratt** [comment]
@john-henry,

I manually fixed it, I host it in a private repo on BitBucket since I don't have a license to share the changes. :/

**Comment by davidbarratt** [comment]
@john-henry,

Honestly I probably violated the license by making any changes in the first place, but I'm certainly not going to share them unless @koken wants to re-license the library.

**Comment by john-henry** [comment]
@davidbarratt 

Thanks David. Not much info out there anymore since SlideshowPro has gone out of business. I'll keep searching then for a bugfix if you are not allowed to share and just suppress errors for now.

Koken is an entirely different thing to the SlideShowPro Director no?

http://slideshowpro.net/

The file classes are for SlideShowPro  and not Koken

**Comment by john-henry** [comment]
Ok I see its the same Brad

**Comment by davidbarratt** [comment]
Yes, @bradleyboy

**Comment by john-henry** [comment]
Actually just fixed it by using the 1.5 api http://wiki.slideshowpro.net/SSPdir/API-DirPHP-Using-15-Beta

Thanks for your time
